### PR TITLE
Run make test in Travis CI build

### PR DESCRIPTION
**Pull request by tsenart**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,4 +9,4 @@ before_install:
 - if [[ "$SrcDir" != "$DestDir" ]]; then mv "$SrcDir" "$DestDir"; cd ../../mesos/$RepoName; export TRAVIS_BUILD_DIR=`dirname $TRAVIS_BUILD_DIR`/$RepoName; fi
 - godep restore
 script: 
-- make
\ No newline at end of file
+- make test
```
Comment (jdef) [inline .travis.yml]: could we change this to `make all test` so that the examples get compiled too?

Comment (tsenart) [inline .travis.yml]: Sure thing.

Comment (tsenart) [inline .travis.yml]: Done.

**Comment by jdef** [comment]
lgtm :ship:

**Pull request merged**